spec/kafka/producers/course_responder_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
```

Comment:
file path should be `spec/responders/courses_spec.rb` and  `spec/responders/sections_spec.rb`

Reply:
Fixed